influxql/ast.go

```diff
@@ -3630,7 +3630,7 @@ func TimeRange(expr Expr) (min, max time.Time, err error) {
 }
 
 // TimeRangeAsEpochNano returns the minimum and maximum times, as epoch nano, specified by
-// and expression. If there is no lower bound, the start of the epoch is returned
+// and expression. If there is no lower bound, the minimum time is returned
```

Comment:
You updated the comment here, but line 3642 still sends the min to `time.Unix(0,0)` when the min time is unset.